packages/three-vrm-core/src/expressions/VRMExpression.ts

```diff
@@ -74,14 +79,25 @@ export class VRMExpression extends THREE.Object3D {
    */
   public get overrideMouthAmount(): number {
     if (this.overrideMouth === 'block') {
-      return 0.0 < this.weight ? 1.0 : 0.0;
+      return 0.0 < this.outputWeight ? 1.0 : 0.0;
     } else if (this.overrideMouth === 'blend') {
-      return this.weight;
+      return this.outputWeight;
     } else {
       return 0.0;
     }
   }
 
+  /**
+   * An output weight of this expression, considering the {@link isBinary}.
+   */
+  public get outputWeight(): number {
+    if (this.isBinary) {
+      return this.weight >= 0.5 ? 1.0 : 0.0;
```

Comment:
```suggestion
      return this.weight > 0.5 ? 1.0 : 0.0;
```

> isBinary : A value greater than 0.5 is 1.0, otherwise 0.0
> https://github.com/vrm-c/vrm-specification/blob/master/specification/VRMC_vrm-1.0/expressions.md#expression-specification

Reply:
Please also modify the test cases.